### PR TITLE
Add mutation from `/foo|bar/` → `/foo/`, `/bar/`

### PR DESCRIPTION
**Pull request by backus**

### CODE REVIEW

**Comment by backus** [comment]
@mbj this is rebased and ready for review

**Comment by backus** [comment]
This still depends on #636 being merged

File: lib/mutant/mutator/node/regexp/alternation_meta.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module Mutant
+  class Mutator
+    class Node
+      module Regexp
+        # Mutator for pipe in `/foo|bar/` regexp
+        class AlternationMeta < Node
+          handle(:regexp_alternation_meta)
+
```
Comment (mbj) [inline lib/mutant/mutator/node/regexp/alternation_meta.rb]: `#dispatch` should normally be private.

Comment (backus) [inline lib/mutant/mutator/node/regexp/alternation_meta.rb]: k

**Pull request merged**